src/main/java/com/basilisk/frontend/views/LoginView.java

```diff
@@ -65,6 +65,9 @@ public void init() {
 
         createUsernameTextField.setMaxLength(15);
         createUsernameTextField.setRequired(true);
```

Comment:
If we don't allow spaces in the create username we shouldn't allow it in the login username. Also the user should have something displayed to them to know they're not allowed